.github/scripts/add-labels.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
You miss Python ! 🤣 

https://github.com/census-instrumentation/opencensus-python/pull/978/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7L5